src/Pages/ProductDetail/RoomInformation/Reservation/Reservation.js

```diff
@@ -0,0 +1,160 @@
+import React, { useState } from "react";
+import styled from "styled-components";
+import { useSelector, useDispatch } from "react-redux";
+import { datesChange } from "../../../../modules/ProducDetail/dayPicker";
+import { DateRangePicker } from "react-dates";
+import moment from "moment";
+import Title from "../../ProductDetailComponent/Title";
+import ReviewInfo from "../../ProductDetailComponent/ReviewInfo";
+import PriceInfo from "./PriceInfo/PriceInfo";
+import mixin from "../../../../Styles/mixin";
+
+export default function Reservation() {
+  const [focusedInput, setFocusedInput] = useState(null);
+
+  const { dateRange } = useSelector(({ dayPicker }) => ({
+    dateRange: dayPicker.dateRange,
+  }));
+  const { startDate, endDate } = dateRange;
+
+  const dateSelected = startDate && endDate;
+
+  const dispatch = useDispatch();
```

Comment:
- 붙여주고 싶네요
```suggestion
  const [focusedInput, setFocusedInput] = useState(null);
  const dispatch = useDispatch();
  const { dateRange } = useSelector(({ dayPicker }) => ({
    dateRange: dayPicker.dateRange,
  }));
  
  const { startDate, endDate } = dateRange;
  const dateSelected = startDate && endDate;
  const title = dateSelected
    ? `${ONEDAY_PRICE} / 박`
    : "요금을 확인하려면 날짜를 입력하세요.";
```
- onDatesChange는 함수니까 변수들 선언과는 분리해주면 좋겠네요